src/convert.js

```diff
@@ -0,0 +1,27 @@
+import JSBI from 'jsbi'
+
+export function getRawRate(convertRate) {
```

Comment:
Until we need it, we probably don't need to export this?

Reply:
Nope, we don't. :)